bandit/core/issue.py

```diff
@@ -127,3 +129,23 @@ def issue_from_dict(data):
     i = Issue(severity=data["issue_severity"])
     i.from_dict(data)
     return i
+
+
+class HashableIssue(object):
+    def __init__(self, issue):
+        '''A hashable issue class cseline mparisions between'''
+        self.issue = issue
+
+    def __hash__(self):
+        issue_hash = 0
+        for prop in ['text', 'severity', 'confidence', 'fname', 'test',
```

Comment:
Taking into account this patch https://github.com/PyCQA/bandit/pull/571, IMO, we don't need to use 'text' here

Reply:
makes sense, changed this.